limesurvey/static/html/limesurvey.html

```diff
@@ -1,6 +1,11 @@
 {% if show_survey %}
 <div class="limesurvey-xblock-wrapper">
+    {% if self.error_message %}
```

Comment:
why this didn't work before?

Reply:
Because Django templates were necessary to be able to include conditionals in the HTML.